projectq/setups/decompositions/phaseestimation_test.py

```diff
@@ -129,7 +128,7 @@ def test_2qubitsPh_andfunction_eigenvectors():
         eng.flush()
 
     num_phase = (results == 0.125).sum()
-    assert num_phase / N >= 0.34, "Statistics phase calculation are not correct (%f vs. %f)" % (
+    assert num_phase / N >= 0.34, "Statistics phase calculation are not correct ({:f} vs. {:f})".format(
```

Comment:
Why not use the `f'{total_prob_after:f}` formatting here?

Reply:
Will be addressed in the next commit.